React-JS-Projects/Advanced/Notes-App/README.md

```diff
@@ -0,0 +1,105 @@
+Users can login and create account using either email/password or Google account.
+      - Authentication: Login and register using either email/password or Google account.
+
+  - Folder Management: Create, edit, delete, and view folders to categorize and store notes.
+
+- Nested Folders: Create nested folders/subfolders, enabling hierarchical organization.
+
+- Note Management: Create, edit, delete, and view notes.
+
+- Text Formatting: Create text with bold, italic, underline, lists, images, and hyperlinks.
+
+- Search Functionality: Easily find notes and folders.
+
+- Responsive Design: Optimized for all screen sizes.
+    </p>
+</div>
+
+
+<!-- -------------------------------------------------------------------------------------------------------------- -->
+
+## :zap: How to run it? 🕹️
+
+To run this project locally, follow these steps:
+
+1. Fork the repository.
+2. Clone the Repository.
+```
+git clone repository-url
+```
+3. go to root of project directory. 
+```
+cd repository-directory
+```
+4. Install Dependencies:
+```
+npm install
+```   
+5. setting up firebase - create .env file and fill the required credentials as below.<br/>
+```
+VITE_FIREBASE_API_KEY=your_api_key
+VITE_FIREBASE_AUTH_DOMAIN=your_auth_domain
+VITE_FIREBASE_PROJECT_ID=your_project_id
+VITE_FIREBASE_STORAGE_BUCKET=your_storage_bucket
+VITE_FIREBASE_MESSAGING_SENDER_ID=your_messaging_sender_id
+VITE_FIREBASE_APP_ID=your_app_id
+```
+6. Run the Development Server.
+```
+npm run dev
+```
+
+<!-- -------------------------------------------------------------------------------------------------------------- -->
+
+## :zap: Screenshots 📸
+<!-- add the screenshot of the project (Mandatory) -->
+<img src="./public/screenshots/image_00.png"/><br/>
+<img src="./public/screenshots/image_01.png"/><br/>
+
+![Line](https://github.com/Avdhesh-Varshney/WebMasterLog/assets/114330097/4b78510f-a941-45f8-a9d5-80ed0705e847)
+
+<!-- -------------------------------------------------------------------------------------------------------------- -->
+
+<h4 align='center'>Developed By <b><i>DHARSHI</i></b> 👦</h4>
+<p align='center'>
+  <a href='https://www.linkedin.com/in/dharshi-balasubramaniyam-47b193243'>
+    <img src='https://img.shields.io/badge/linkedin-%230077B5.svg?style=for-the-badge&logo=linkedin&logoColor=white' />
+  </a>
+  <a href='https://github.com/DharshiBalasubramaniyam'>
+    <img src='https://img.shields.io/badge/github-%23121011.svg?style=for-the-badge&logo=github&logoColor=white' />
+  </a>
+</p>
+
+<h4 align='center'>Happy Coding 🧑‍💻</h4>
+
+<h3 align="center">Show some &nbsp;❤️&nbsp; by &nbsp;🌟&nbsp; this repository!</h3>
+    <img src='https://img.shields.io/badge/github-%23121011.svg?style=for-the-badge&logo=github&logoColor=white' />
+  </a>
+</p>
+
+<h4 align='center'>Happy Coding 🧑‍💻</h4>
+
```

Comment:
![image](https://github.com/Avdhesh-Varshney/WebMasterLog/assets/114330097/083345bb-1a1d-4e8a-8956-37be08f526f0)

2 times repetation of 1 thing. You have to update your data in template only not copy and paste it 2 times.

Reply:
Now I have made required changes in read me file.